src/components/TopBar.tsx

```diff
@@ -8,11 +8,41 @@ import TopBarSearch from './TopBarSearch';
 
 export default function TopBar() {
     const [blockHeight, setBlockHeight] = useState('...');
+    const [totalTransactions, setTotalTransactions] = useState('...');
+    const [averageTxPerSecond, setAverageTxPerSecond] = useState('...');
+    const [hashRate, setHashRate] = useState('...');
+    const [averageFee, setAverageFee] = useState('...');
+    const [averageBlockTime, setAverageBlockTime] = useState('...');
 
     const loadMetadata = useCallback(async () => {
+        const meta = {
+            "chainRunningTime": {
+                "runningTimeMillis": 2992858000
+            },
+            "hashRate": "3.5",
+            "averageFee": "0.05",
+            "averageBlockTime": "215"
+        };
         const metadata = await fetchChainMetadata();
-        const formattedBlockHeight = numeral(metadata.blockHeight).format('0,0');
+
+        const formattedBlockHeight = numeral(metadata.blockHeight).format('0.0a');
         setBlockHeight(formattedBlockHeight);
+
+        const formattedTotalTransactions = numeral(metadata.totalTransactions).format('0.0a');
+        setTotalTransactions(formattedTotalTransactions);
+
+        const calcAverageTxPerSecond = metadata.totalTransactions / (meta.chainRunningTime.runningTimeMillis / 1000);
```

Comment:
You don't need to use the chain running time anymore. The API is returning everything you need.